### PR TITLE
musescore-2.0.0.ebuild by Artur Wrona

### PR DESCRIPTION
**Pull request by yurats**
Sorry for re-opened pull request -- I mistakenly added other commits in the previous request.

 Newline at EOF fixed.

### CODE REVIEW

File: media-sound/musescore/musescore-2.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "
+
+DEPEND="${RDEPEND}
+    dev-texlive/texlive-context
+    app-doc/doxygen
+    virtual/pkgconfig"
+
+S="${WORKDIR}/${MY_P}"
+
+
```
Comment (jlec) [inline media-sound/musescore/musescore-2.0.0.ebuild:39]: Please check your ebuilds using repoman. Additional empty line here.

**Comment by jlec** [comment]
does qmake get called? If yo you need to make sure the Qt5 version gets used.

**Comment by mgorny** [comment]
Next time, you don't have to open new pull request. Just `git push --force` to the original branch, and github will update the old PR.

**Comment by yurats** [comment]
Hmm... May be I can make working ebuild later, sorry:/

**Pull request closed without merging**